docs/contributing-to-analyses/pr-review-and-merge/respond-to-review.md

```diff
@@ -0,0 +1,167 @@
+# Responding to code reviews
+
+Code review is essentially a conversation between you and your reviewer.
+To ensure that this process goes smoothly and with fewer opportunities for miscommunication, we strongly recommend reading through all of your reviewer's comments before diving into review:
+
+- You'll gain more context about why the reviewer left certain comments by reading the whole review first.
+    - Several comments may also be related to each other, and taking in the full review will help you identify similar concepts or areas the reviewer noted.
+- It's also possible that your reviewer misunderstood, or differently interpreted, some changes you made.
+    - Reading the full review will help you understand if there are areas that need clarification before you can dive into responding.
+
+## Navigating the pull request page
+
+On the top of the PR page, you will see four tabs:
+
+<figure markdown="span">
+    ![Pull request tabs.](../../img/respond-to-review-tabs.png){width="800"}
+</figure>
+
+<div class="grid" markdown>
+
+
+- **`Conversation`**
+    - This tab shows the main page of the PR.
+    Here, you can find the original description of the PR along with any comments left on the PR, both reviewer comments and any comments you leave in response.
+    - You can also find any [resolved comments](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#resolving-conversations) here, if you need to refer to them again.
+- **`Commits`**
+    - This tab shows a list of all commits added to the branch that is being reviewed.
+    Your reviewer may use this page to explore changes in individual commits.
+
+<!-- comment to force two grid columns -->
+
+- **`Checks`**
+    - This tab shows results from certain types of code checks that have been automatically set up by the Data Lab; you can ignore this tab.
+- **`Files changed`**
+    - This tab shows a line-by-line overview of content you have added and removed in your branch.
+    Changes shown here should reflect the difference between files in your branch and the base branch your PR is targeting.
+This should be the `main` branch of the `AlexsLemonade/OpenScPCA-analysis` repository.
+    - Referring to this tab is a great way to quickly see all your changes, as well as respond to [reviewer suggestions or comments](#inline-comments).
+
+</div>
+
+
+## Responding to review comments
+
+When making code changes in response to review, you need to make sure your local repository is staying in-sync with any changes made directly on GitHub.
```

Comment:
```suggestion
In some cases a reviewer will request you make changes without providing in-line suggestions. 
When this happens, you will need to make the code changes locally. 
If you committed suggestions directly on GitHub before working on additional changes, be sure to sync your local branch with changes made on GitHub before changing any local code. 
```
Just expanding on when they would actually need this. 
Also I don't think this section makes any sense here and I would put this after the `Types of reviewer comments`.